### PR TITLE
Provide Text for TODOs in Comments

### PR DESCRIPTION
**Pull request by dvonthenen**
Addresses issue: https://github.com/deepgram/deepgram-dotnet-sdk/issues/210

This PR just fills in `TODO` placeholders for comments. There should be zero code changes in here (that I can recall).

## Summary by CodeRabbit

- **Documentation**
	- Enhanced comments and documentation for various classes and records in different namespaces to clarify functionalities related to text processing, live audio processing, management operations, and pre-recorded audio processing.
	- Improved property descriptions for sentiment analysis, intents, topics, and more to provide better insights for end-users and developers.

### CODE REVIEW

File: Deepgram/Clients/Live/v1/Client.cs [hunk 1]
```diff
@@ -308,7 +308,8 @@ internal async Task ProcessSendQueue()
                 Log.Verbose("ProcessSendQueue", "Reading message of queue...");
                 while (_sendChannel.Reader.TryRead(out var message))
                 {
-                    Log.Verbose("ProcessSendQueue", $"Sending message..."); // TODO: dump this message
+                    // TODO: Add logging for message capturing for possible playback
+                    Log.Verbose("ProcessSendQueue", $"Sending message...");
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply Deepgram/Clients/Live/v1/Client.cs:312]: Ignoring... I know what the comment means.

File: Deepgram/Factory/HttpClientFactory.cs [hunk 2]
```diff
@@ -46,7 +46,6 @@
                 client.DefaultRequestHeaders.Add(header.Key, header.Value);
             }
 
-        // TODO: log
         client.BaseAddress = new Uri(options.BaseAddress);
 
         return client;
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply Deepgram/Factory/HttpClientFactory.cs:51]: Sound good!

File: Deepgram/Models/PreRecorded/v1/Average.cs [hunk 3]
```diff
@@ -7,13 +7,13 @@
...
     /// <summary>
-    /// TODO
+    /// Sentiment: positive, negative, neutral
     /// </summary>
     [JsonPropertyName("sentiment")]
     public string? Sentiment { get; set; }
 
     /// <summary>
-    /// TODO
+    /// Sentiment score
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply Deepgram/Models/PreRecorded/v1/Average.cs:16]: ok!

File: Deepgram/Models/Analyze/v1/Average.cs [hunk 4]
```diff
@@ -7,13 +7,13 @@ namespace Deepgram.Models.Analyze.v1;
 public record Average
 {
     /// <summary>
-    /// TODO
+    /// Sentiment of the text segment.
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply Deepgram/Models/Analyze/v1/Average.cs:10]: ok!

File: Deepgram/Models/PreRecorded/v1/Average.cs [hunk 5]
```diff
@@ -7,13 +7,13 @@ namespace Deepgram.Models.PreRecorded.v1;
 public record Average
 {
     /// <summary>
-    /// TODO
+    /// Sentiment: positive, negative, neutral
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply Deepgram/Models/PreRecorded/v1/Average.cs:10]: We get the general idea... dont need get into super details. There is documentation on this.

File: Deepgram/Models/PreRecorded/v1/Average.cs (hunk 3 above)
[the remark this answers is not in the document]
Comment (dvonthenen) [reply Deepgram/Models/PreRecorded/v1/Average.cs]: We get the general idea... dont need get into super details. There is documentation on this.

File: Deepgram/Models/Analyze/v1/Average.cs [hunk 6]
```diff
@@ -7,13 +7,13 @@
...
     /// </summary>
     [JsonPropertyName("sentiment")]
     public string? Sentiment { get; set; }
 
     /// <summary>
-    /// TODO
+    /// Sentiment score of the text segment.
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply Deepgram/Models/Analyze/v1/Average.cs]: We get the general idea... dont need get into super details. There is documentation on this.

File: Deepgram/Models/OnPrem/v1/CredentialResponse.cs [hunk 7]
```diff
@@ -7,13 +7,13 @@ namespace Deepgram.Models.OnPrem.v1;
 public record CredentialResponse
 {
     /// <summary>
-    /// TODO
+    /// Member information
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply Deepgram/Models/OnPrem/v1/CredentialResponse.cs:10]: We get the general idea... dont need get into super details. There is documentation on this.

File: Deepgram/Models/OnPrem/v1/CredentialResponse.cs [hunk 8]
```diff
@@ -7,13 +7,13 @@
...
     /// </summary>
     [JsonPropertyName("member")]
     public Member? Member { get; set; }
 
     /// <summary>
-    /// TODO
+    /// Distribution credentials
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply Deepgram/Models/OnPrem/v1/CredentialResponse.cs:16]: We get the general idea... dont need get into super details. There is documentation on this.

File: Deepgram/Models/PreRecorded/v1/ModelInfo.cs [hunk 9]
```diff
@@ -7,19 +7,19 @@ namespace Deepgram.Models.PreRecorded.v1;
 public record ModelInfo
 {
     /// <summary>
-    /// TODO
+    /// Architecture
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply Deepgram/Models/PreRecorded/v1/ModelInfo.cs]: We get the general idea... dont need get into super details. There is documentation on this.

File: Deepgram/Models/Manage/v1/Callback.cs [hunk 10]
```diff
@@ -7,19 +7,19 @@
...
     /// <summary>
-    /// TODO
+    /// Return Code
     /// </summary>
     [JsonPropertyName("code")]
     public int? Code { get; set; }
 
     /// <summary>
-    /// TODO
+    /// Is Completed
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply Deepgram/Models/Manage/v1/Callback.cs:22]: We get the general idea... dont need get into super details. There is documentation on this.

File: Deepgram/Models/Manage/v1/Callback.cs [hunk 11]
```diff
@@ -7,19 +7,19 @@ namespace Deepgram.Models.Manage.v1;
 public record Callback
 {
     /// <summary>
-    /// TODO
+    /// Attempt number.
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply Deepgram/Models/Manage/v1/Callback.cs:10]: We get the general idea... dont need get into super details. There is documentation on this.

File: Deepgram/Models/Manage/v1/UsageRequestsResponse.cs [hunk 12]
```diff
@@ -7,19 +7,19 @@ namespace Deepgram.Models.Manage.v1;
 public record UsageRequestsResponse
 {
     /// <summary>
-    /// TODO
+    /// Page number
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply Deepgram/Models/Manage/v1/UsageRequestsResponse.cs:10]: We get the general idea... dont need get into super details. There is documentation on this.

File: Deepgram/Models/Manage/v1/UsageRequestsResponse.cs [hunk 13]
```diff
@@ -7,19 +7,19 @@
...
     /// </summary>
     [JsonPropertyName("page")]
     public int? Page { get; set; }
 
     /// <summary>
-    /// TODO
+    /// Results per page
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply Deepgram/Models/Manage/v1/UsageRequestsResponse.cs:16]: We get the general idea... dont need get into super details. There is documentation on this.

File: Deepgram/Models/Manage/v1/UsageRequestsResponse.cs [hunk 14]
```diff
@@ -7,19 +7,19 @@
...
     /// </summary>
     [JsonPropertyName("limit")]
     public int? Limit { get; set; }
 
     /// <summary>
-    /// TODO
+    /// Requests response/result
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply Deepgram/Models/Manage/v1/UsageRequestsResponse.cs:22]: We get the general idea... dont need get into super details. There is documentation on this.

File: Deepgram/Models/Analyze/v1/SummaryInfo.cs [hunk 15]
```diff
@@ -7,19 +7,19 @@ namespace Deepgram.Models.Analyze.v1;
 public record SummaryInfo
 {
     /// <summary>
-    /// TODO
+    /// Input tokens used in the model.
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply Deepgram/Models/Analyze/v1/SummaryInfo.cs:10]: We get the general idea... dont need get into super details. There is documentation on this.

File: Deepgram/Models/Analyze/v1/SummaryInfo.cs [hunk 16]
```diff
@@ -7,19 +7,19 @@
...
     /// </summary>
     [JsonPropertyName("input_tokens")]
     public int? InputTokens { get; set; }
 
     /// <summary>
-    /// TODO
+    /// Model UUID.
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply Deepgram/Models/Analyze/v1/SummaryInfo.cs:16]: We get the general idea... dont need get into super details. There is documentation on this.

File: Deepgram/Models/Analyze/v1/SummaryInfo.cs [hunk 17]
```diff
@@ -7,19 +7,19 @@
...
     /// </summary>
     [JsonPropertyName("model_uuid")]
     public string? ModelUUID { get; set; }
 
     /// <summary>
-    /// TODO
+    /// Output tokens used in the model.
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply Deepgram/Models/Analyze/v1/SummaryInfo.cs:22]: We get the general idea... dont need get into super details. There is documentation on this.

File: Deepgram/Models/Analyze/v1/TopicsInfo.cs [hunk 18]
```diff
@@ -7,19 +7,19 @@ namespace Deepgram.Models.Analyze.v1;
 public record TopicsInfo
 {
     /// <summary>
-    /// TODO
+    /// Input tokens used in the model.
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply Deepgram/Models/Analyze/v1/TopicsInfo.cs:10]: We get the general idea... dont need get into super details. There is documentation on this.

File: Deepgram/Models/Analyze/v1/TopicsInfo.cs [hunk 19]
```diff
@@ -7,19 +7,19 @@
...
     /// </summary>
     [JsonPropertyName("input_tokens")]
     public int? InputTokens { get; set; }
 
     /// <summary>
-    /// TODO
+    /// Model UUID.
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply Deepgram/Models/Analyze/v1/TopicsInfo.cs:16]: We get the general idea... dont need get into super details. There is documentation on this.

File: Deepgram/Models/Analyze/v1/TopicsInfo.cs [hunk 20]
```diff
@@ -7,19 +7,19 @@
...
     /// </summary>
     [JsonPropertyName("model_uuid")]
     public string? ModelUuid { get; set; }
 
     /// <summary>
-    /// TODO
+    /// Output tokens used in the model.
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply Deepgram/Models/Analyze/v1/TopicsInfo.cs:22]: We get the general idea... dont need get into super details. There is documentation on this.

**Review by lukeocodes — DISMISSED** [review]

**Review by jpvajda — APPROVED** [review]
LGTM

**Pull request merged**